src/EFCore.PG/Storage/Internal/NpgsqlDatabaseCreator.cs

```diff
@@ -319,7 +319,8 @@ public override void CreateTables()
             // If a PostgreSQL extension or enum was added, we want Npgsql to reload all types at the ADO.NET level.
             var reloadTypes = operations.Any(o => o is AlterDatabaseOperation &&
                 (PostgresExtension.GetPostgresExtensions(o).Any() ||
-                 PostgresEnum.GetPostgresEnums(o).Any()));
+                 PostgresEnum.GetPostgresEnums(o).Any() ||
```

Comment:
Can you bring the previous condition down one line so they are all aligned?

Reply:
Changed the alignment.